docs/release/release-2.0.rst

```diff
@@ -0,0 +1,40 @@
+===========
+Release 2.0
+===========
+
+New Features
+------------
+* Make presto-admin log and configuration directories configurable. They can be
+  set using the environment variables ``PRESTO_ADMIN_LOG_DIR`` and
+  ``PRESTO_ADMIN_CONFIG_DIR``.
+* Change the default configuration directory to ``~/.prestoadmin`` and the
+  default log directory to ``~/.prestoadmin/log``
+* Remove requirement for running and installing presto-admin with sudo.
+  Presto-Admin still uses sudo to execute commands on the Presto nodes.
```

Comment:
I was trying to say that presto-admin still uses sudo internally.  I can just leave that bit out if it's more confusing than helpful